### ISSUE TITLE
Can't get https to work

### ISSUE THREAD

**Issue by threehappypenguins** [datarhei/restreamer#443]
Sorry I opened another ticket. I can't figure out how to "unclose" the old one.

I just installed Restreamer on a GCP instance without issue. In the GCP firewall, I have opened ports 8080 and 8181 (and also 80 and 443 by default), and via SSH I have added the following to my iptables:

```
sudo iptables -t nat -I PREROUTING -p tcp --dport 80 -j REDIRECT --to-ports 8080
sudo iptables -t nat -I OUTPUT -p tcp -o lo --dport 80 -j REDIRECT --to-ports 8080
sudo iptables -t nat -I PREROUTING -p tcp --dport 443 -j REDIRECT --to-ports 8181
sudo iptables -t nat -I OUTPUT -p tcp -o lo --dport 443 -j REDIRECT --to-ports 8181
```

I can see my rules when I do:

`sudo iptables -t nat -L`

I can access the ui without issue by going to http://my.domain.com/ui

In the ui, I followed the instructions in the quickstart video by going to Settings > Network and then enabled https just like in the video. But I still can't access the ui either by https://my.domain.com/ui or https://my.domain.com (ERR_CONNECTION_CLOSED then changes to ERR_CONNECTION_RESET). https://my.domain/com:8181 and https://my.domain/com:8181/ui and https://my.domain/com:8181 produces ERR_CONNECTION_REFUSED so I'm guessing I shouldn't/can't access with the 8181 port in the address.

There's probably something very simple that I'm missing and not understanding. Please advise.

**Comment by threehappypenguins** [comment]
An additional, but related question. I had to delete my redirect rule for port 443 to 8181 in order to restream to YouTube which uses rtmps (and therefore port 443). It would not connect and restream unless I deleted the rule. How can I have a secure ssl ui site for restreamer unless I redirect port 443 to 8181 and simultaneously be able to restream with rtmps?

**Comment by threehappypenguins** [comment]
I was able to get somewhere when I read [this post](https://github.com/datarhei/restreamer/issues/430#issuecomment-1282663666) here on changing 8181 to 443 for the docker container itself (443:443).

In the UI, I changed it from 8181 to 443, and then opened the log to watch what happened (`docker logs --follow <container ID>`). I saw a certificate was created and all was well, and I could access the site at https://my.domain.com/ui

I'm not sure if this is a secure way of doing things. Can anyone advise?

At this point, I am now trying to figure out how to redirect http to https so that http cannot be used.
